### ISSUE TITLE
devd lots of messages to syslog

### ISSUE THREAD

**Issue by tsumaru720** [opnsense/core#1484]
Recently upgraded to 17.1 and I've noticed my remote syslog server is getting about 80 messages per minute from devd

My OPNsense install runs in a VM 

```
devd: Processing event '!system=CAM subsystem=periph type=error device=cd0 serial="QM00003" cam_status="0xcc" scsi_status=2 scsi_sense="70 02 3a 00" CDB="00 00 00 00 00 00 " '
devd: Processing notify event
devd: Popping table
devd: Pushing table
devd: Pushing table
devd: Processing notify event
devd: Popping table
devd: Processing event '!system=CAM subsystem=periph type=error device=cd0 serial="QM00003" cam_status="0xcc" scsi_status=2 scsi_sense="70 02 3a 00" CDB="00 00 00 00 00 00 " '
```
etc is being put into syslog
FWIW this VM has a CD drive attached, but no media is attached

I've tried putting in /etc/rc.conf.d/devd
```
devd_flags="-q" 
```
but it doesnt seem to apply, neither does devd_enable="NO"

For now im just having to kill the process whenever the VM boots but some suggestions on how to quell this would be nice

Disclaimer: I'm not all too familiar with BSD

**Comment by tsumaru720** [comment]
For now I have worked around the issue by removing the cdrom device from the virtual machine - But im not sure why devd was complaining as the drive was attached to the VM but empty :)

**Comment by fichtner** [comment]
Devd is started from here: https://github.com/opnsense/core/blob/master/src/etc/inc/system.inc#L1549

There is definitely something wrong with CAM and drives as we e.g. know that BSNMPD goes haywire when a CD drive is attached, but no CD inserted. Maybe that's the same issue. Does it also happen when you mount a CD?

**Comment by tsumaru720** [comment]
@fichtner no - If I have a CD drive attached with a CD "mounted" the issue is not present - just when an empty drive exists.

**Comment by fichtner** [comment]
I think it's the same as:

https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=217983
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=215471

I'm watching both bugs, will tag upstream for now... scope is beyond us

**Comment by fichtner** [comment]
Waiting for upstream, no use keeping this open

**Comment by fichtner** [comment]
Uh no, the 100% cpu in snmp hostres is fixed in 18.1 :)
